web-client/src/presenter/errors/ErrorFactory.test.js

```diff
@@ -43,6 +43,13 @@ describe('ErrorFactory', () => {
     expect(result.className).toEqual('ServerInvalidResponseError');
     expect(result.title).toEqual('An error has occurred');
   });
+  it('creates GatewayTimeoutError errors for status code 504', () => {
+    const error = new Error();
+    error.response = { data: 'Unauthorized', status: 504 };
```

Comment:
I think I might change the text for `Unauthorized` to something more representative of what's happening. Seems like `Gateway Timeout` would be most apt.

Reply:
That's just random test data, any string will work.